### DISCUSSION TITLE
Get clicked menu item on MenuButton

### DISCUSSION THREAD

**Discussion by blokpardi** [microsoft/fluentui#24942] [General]
I have a menu button, it renders fine and I can fire a function with the onMenuItemClick event. But I can't for the life of me figure out how to get the item that was clicked on? I don't see anything in the props. It's driving me mad, can someone help?

**Comment by micahgodbolt** [comment]
Can you create a small codepen demo of this issue so that we can fork it and try to provide an answer?
